tests/utils/dockercli/volumelifecycle.go

```diff
@@ -25,16 +25,15 @@ import (
 	"github.com/vmware/docker-volume-vsphere/tests/utils/ssh"
 )
 
-// CreateDefaultVolume is going to create vsphere docker volume with
-// defaults.
-func CreateDefaultVolume(ip, name string) ([]byte, error) {
+// CreateVolume is going to create vsphere docker volume with given name.
+func CreateVolume(ip, name string) ([]byte, error) {
```

Comment:
how about `CreateVolumeWithDefaults`? inshort .. it would be better to distinguish the volume creation method name. 

Please share your idea on the same.